DOCS.md

```diff
@@ -0,0 +1,29 @@
+Use this plugin to publish artifacts from the build to an SFTP server
+You can override the default configuration with the following parameters:
+
+* `host` - Server host
+* `port` - Server port, default to 22
+* `username` - Server username, default to blank
+* `password` - Password for password-based authentication
+* `privateKey` - String containing the private key for key-based authentication
+* `privateKeyFile` - Path to a private key (Used in conjonction with `volumes`) for key-based authentication
```

Comment:
good catch from @tboerger  ... do you mind if we stick to snake case for consistency with the other plugins?